ci/ray_ci/bisect/linux_validator.py

```diff
@@ -0,0 +1,83 @@
+import time
+
+from pybuildkite.buildkite import Buildkite
+
+from ci.ray_ci.bisect.validator import Validator
+from ci.ray_ci.utils import logger
+from ray_release.test import (
+    Test,
+    LINUX_TEST_PREFIX,
+)
+from ray_release.aws import get_secret_token
+from ray_release.configs.global_config import get_global_config
+
+BUILDKITE_ORGANIZATION = "ray-project"
+BUILDKITE_POSTMERGE_PIPELINE = "postmerge"
+BUILDKITE_BUILD_RUNNING_STATE = [
+    "creating",
+    "scheduled",
+    "running",
+]
+BUILDKITE_BUILD_PASSING_STATE = [
+    "passed",
+    "skipped",
+]
+BUILDKITE_BUILD_FAILING_STATE = [
+    "failing",
+    "failed",
+    "blocked",
+    "canceled",
+    "canceling",
+    "not_run",
+]
```

Comment:
are these linux validator specific? or is it that macos is not using this?

I am a bit worrying that this is freezing how tests can run on the postmerge pipeline with a bisector, or making this bisector very easy to break and left unmaintained overtime.

I feel that we would need to have a clearer spec on what is the desired way to run a subset of tests on the pipeline first? like is there any hope / roadmap that this `LinuxValidator` is going to be more generic/reusable?

or maybe it is just a naming thing? like this file/class should be named as `GeneralValidator`? or `BuildkiteTestJobValidator` or just `TestJobValidator`?

Reply:
it's just the naming thing; the only linux about this logic is line 47 [1]; i'm adding a WindowsValidator in another PR and in that PR i will split this into `Linux/WindowsValidator` to inherits from the `GeneralValidator`

Reply:
ok. then maybe just rename this to `GenericValidator` or `BuildkiteValidator` or something? so that it is not associated to Linux, where the `LINUX_TEST_PREFIX` is the default value of an arg? I just feels that all the logic here is really mostly generic, but not related to linux. Like the `RAYCI_SELECT` part does not really care if it is Linux or not.